### PR TITLE
Upgrade Workbox to v4.0.0

### PR DESCRIPTION
**Pull request by westonruter**
Build of rc0 for testing: [pwa.zip](https://github.com/xwp/pwa-wp/files/2838682/pwa.zip)

Build of rc2 for testing: [pwa.zip](https://github.com/xwp/pwa-wp/files/2866214/pwa.zip)

### CODE REVIEW

**Comment by westonruter** [comment]
@aristath Would you please smoke test this on one of your project?

File: package.json [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
   "grunt-wp-deploy": "^1.3.0",
   "husky": "^0.14.3",
-  "workbox-cli": "4.0.0-beta.2",
+  "workbox-cli": "4.0.0-rc.2",
   "wp-dev-lib": "github:xwp/wp-dev-lib#1.0.0"
  },
  "scripts": {
   "install": "composer install",
-  "precommit": "./node_modules/wp-dev-lib/scripts/pre-commit",
+  "precommit": "if command -v lando >/dev/null 2>&1; then lando ssh -c 'bash ./node_modules/wp-dev-lib/scripts/pre-commit'; else ./node_modules/wp-dev-lib/scripts/pre-commit; fi",
```
Comment (westonruter) [inline package.json]: @felixarntz what do you think of this?

Comment (felixarntz) [reply package.json]: I guess that would work, but tbh I'm really not a fan of putting environment-specific details into a general project repository, unless that environment has something to do with standardization (e.g. `editorconfig`).

Comment (westonruter) [reply package.json]: Struggling to think of an alternative way to keep Git running on the host machine but to run all Git hooks in the container.

**Comment by westonruter** [comment]
Updated to 4.0.0 which was just released.

**Pull request merged**